docs/configure-artifact-repository.md

```diff
@@ -373,25 +359,98 @@ artifacts:
       endpoint: https://mystorageaccountname.blob.core.windows.net
       container: my-container-name
       blob: path/in/container
-      # accountKeySecret is a secret selector.
-      # It references the k8s secret named 'my-azure-storage-credentials'.
-      # This secret is expected to have have the key 'account-access-key',
-      # containing the base64 encoded credentials to the storage account.
-      #
       # If a managed identity has been assigned to the machines running the
       # workflow (e.g., https://docs.microsoft.com/en-us/azure/aks/use-managed-identity)
-      # then accountKeySecret is not needed, and useSDKCreds should be
-      # set to true instead:
-      # useSDKCreds: true
-      accountKeySecret:
-        name: my-azure-storage-credentials
-        key: account-access-key     
+      # then useSDKCreds should be set to true. The accountKeySecret is not required
+      # and will not be used in this case.
+      useSDKCreds: true  
 ```
 
-If `useSDKCreds` is set to `true`, then the `accountKeySecret` value is not
-used and authentication with Azure will be attempted using a
-[`DefaultAzureCredential`](https://docs.microsoft.com/en-us/azure/developer/go/azure-sdk-authentication)
-instead.
+In addition to managed identities, Argo workflows also support authentication using access keys and SAS tokens.
```

Comment:
```suggestion
You can also authenticate with [Access Keys](#using-azure-access-keys) and [Shared Access Signatures](#using-azure-shared-access-signatures].
```

- simplify, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
- use active voice, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice)
- Access Keys is a proper noun, no?

- also link to the two sections and use the full name for SAS as it has not been introduced yet.

Reply:
Thanks @agilgur5 ! I'll raise a separate PR for this!

Reply:
Addressed in #13376